.rubocop_styleguide.yml

```diff
@@ -196,6 +196,7 @@ Metrics/ClassLength:
   Max: 100
   Exclude:
   - engines/order_management/app/services/order_management/reports/enterprise_fee_summary/scope.rb
+  - lib/open_food_network/users_and_enterprises_report.rb
```

Comment:
we need to fix rubocop problems and add local exceptions if really needed.
this line should not be here.

Reply:
oh, I just saw @kristinalim comment.
now I am confused, I am not sure what's the current usage of the different rubocop exception files and when we should use local exceptions like [this](https://github.com/openfoodfoundation/openfoodnetwork/blob/59d4e3d9e2192524d807b8dbc4271b427adea1ac/lib/tasks/sample_data/enterprise_factory.rb#L20).

Reply:
me and @mkllnk talked about this and this is what we have agreed:
https://community.openfoodnetwork.org/t/management-of-rubocop-exception-files/1668
maybe we can agree on this and make this PR follow the rule agreed.
our current proposed approach is to:
- add local exception for this problem
- or, if it needs to be fixed at some point, add to the manual_todo list

Reply:
Thanks for bringing this to my attention, @luisramos0! This sounds good to me - thanks.